.github/workflows/release.yml

```diff
@@ -47,12 +46,25 @@ jobs:
           # want >=2 
           podman --version
 
-          # do we need all these?
+          # do we need any of these?
           # docker-ce          x86_64  3:19.03.14-3.el7            docker-ce-stable   24 M
           # gcc-c++            x86_64  4.8.5-44.el7                base              7.2 M
-          # bash >=5
           # nodejs             x86_64  2:10.23.0-1nodesource       nodesource         20 M
           # yarn               noarch  1.22.5-1                    yarn              1.2 M
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login to docker.io
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
@benoitf do you have access to the docker and quay secrets and can you add them to this repo? I'm not admin for some reason :( and also I don't have the secrets...